src/components/Button/Button.tsx

```diff
@@ -18,7 +18,7 @@ export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
   /**
    * Is this the principal call to action on the page?
    */
-  buttonType?: 'primary' | 'secondary' | 'ghost';
+  buttonType?: 'primary' | 'secondary' | 'ghost' | 'icon';
```

Comment:
it would be nice if we renamed this to `variant` while we were in here... :|

Reply:
this is a little bit confusing with the `iconLast` options above...  I guess, do we need that prop still?  Maybe we can ask David what he thinks...  I also think it's a bit confusing to have the story be button with icon.

Reply:
I like renaming to variant. Open to suggestions on icon placement prop.